From f2e93e1fd01c27a4c20dde0fe9dca1d0301c14ea Mon Sep 17 00:00:00 2001
From: Etienne Carriere <etienne.carriere@foss.st.com>
Date: Thu, 15 Feb 2024 07:19:03 +0100
Subject: [PATCH 0729/1695] plat-stm32mp1: default enable SAES software
 fallback

Default enable SAES software fallback for 192bit keys support.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git c5bd5a7c6856dc89a1d6f966051a864737b6c573 plat-stm32mp1: default enable SAES software fallback]
Reviewed-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Signed-off-by: Etienne Carriere <etienne.carriere@foss.st.com>
Change-Id: Ic97f61e729813628f424ed01aaf91bc06e6d520a
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/372867
Tested-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 core/arch/arm/plat-stm32mp1/conf.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index ccc3105df..f78a8f136 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -440,6 +440,9 @@ CFG_WERROR ?= y
 CFG_DRIVERS_CLK_PRINT_TREE ?= $(CFG_WITH_STATS)
 CFG_DRIVERS_REGULATOR_PRINT_TREE ?= $(CFG_WITH_STATS)
 
+# Default enable software fallback on crypto drivers
+CFG_STM32_SAES_SW_FALLBACK ?= y
+
 # Enable OTP update with BSEC driver
 CFG_STM32_BSEC_WRITE ?= y
 
-- 
2.47.2

